wizards/Argo/ArgoWizard.tsx

```diff
@@ -301,9 +301,16 @@ export function ArgoWizard(props: ArgoWizardProps) {
                                 options={gitChannels}
                                 onValueChange={(value) => {
                                     const channel = props.channels?.find((channel) => channel.spec.pathname === value)
-                                    if (channel) {
-                                        setGitRevisionsAsyncCallback(() => () => getGitBranchList(channel, props.getGitRevisions))
-                                    }
```

Comment:
If a newly created channel is invalid, we want to deal with the within the async callback, so we can clear the values in the Branch field.